packages/ncids-js/src/ncids-full.ts

```diff
@@ -0,0 +1,13 @@
+/**
```

Comment:
These should be auto inits so that someone can just add thse to their page. I think we should exclude anything without a normal autoinit and make a backlog for it.